sizeOptimizer.cpp

```diff
@@ -238,6 +363,27 @@ int SizeOptimizer::getNoOfPrimes(Polynom &poly) {
   if (poly.exponent3 != 0) counter++;
   if (poly.exponent5 != 0) counter++;
   if (poly.exponent7 != 0) counter++;
+  if (poly.exponent11 != 0) counter++;
+  return counter;
+}
+
+int SizeOptimizer::getNoOfPrimes(long size) {
```

Comment:
this method gives wrong answer for `15`:
The Prime Factorization is 3 x 5 (e.g. `2`), method returns `3`